### PR TITLE
list_users function done

### PR DESCRIPTION
**Pull request by kelliartamas**

### CODE REVIEW

**Comment by kotoszo** [comment]
nice job dude

File: templates/users.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %} {% block content %}
+<div id="body-users" class="container">
+    {% for user in users %}
+    <table class="table-form">
```
Comment (jpeter82) [inline templates/users.html]: Please add a table header. Thanks.

File: templates/users.html [hunk 2]
```diff
@@ -1,7 +1,18 @@
 {% extends "base.html" %} {% block content %}
 <div id="body-users" class="container">
-    {% for user in users %}
     <table class="table-form">
+        {% for user in users %}
```
Comment (kotoszo) [inline templates/users.html]: such a noob

**Comment by jpeter82** [comment]
Looks good :-)

**Pull request merged**